pynumdiff/finite_difference/_finite_difference.py

```diff
@@ -68,52 +50,32 @@ def second_order(x, dt):
     return x, dxdt_hat
 
 
-def __x_hat_using_finite_difference__(x, dt):
-    """
-    :param x:
-    :param dt:
-    :return:
+def _x_hat_using_finite_difference(x, dt):
+    """Find a smoothed estimate of the true function by taking FD and then integrating with trapezoids
```

Comment:
Is my interpretation correct here? Why differentiate only to immediately integrate?